### ISSUE TITLE
Allow sighted users to enable accessibility without a screen reader

### ISSUE THREAD

**Issue by marcianx** [flutter/flutter#125415]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/flutter/flutter/issues)
- [X] I have read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)

### Use case

Sighted users might be using non-screen-reader assistive tools supporting text-to-speech and speech-to-text tools as Chrome plugins, but these tools see nothing since no DOM is being emitted.

The problem is that there's no (practical) way for a sighted non-screen reader user to enable accessibility. The user may tab into the invisible button, but they get no visual feedback that they are on it and can activate it. We need something we can document for end users.

### Proposal

* It might be helpful if the "Enable accessibility" button became actually visible when focused.
* Maybe it could also be enabled using some hotkey combination which when triggered shows a visual confirmation that accessibility was enabled.

**Comment by goderbauer** [comment]
Labeling this as platform-web since this button is not produced by the framework.

You can always manually incorporate such a button in your UI and have it call SemanticsBinding.instance.ensureSemantics() to turn semantics on. (dispose the returned SemanticsHandle to turn it off again).

**Comment by yjbanov** [comment]
We provide an API for enabling accessibility:

```dart
SemanticsBinding.instance.ensureSemantics();
```

So, for example, you can create a button to enable it, or enable it based on some app settings.

Is there anything else we can do to make it better?

**Comment by marcianx** [comment]
Sorry, I should have responded earlier. I suppose this could be a viable workaround.

**Comment by yjbanov** [comment]
Cool. I'll close the issue for now, but if something in Flutter prevents you from using the suggested workaround, feel free to file a new issue.
